dmscripts/notify_suppliers_of_new_questions_answers.py

```diff
@@ -0,0 +1,159 @@
+import dmapiclient
+
+from datetime import datetime, date, timedelta
+
+from dmscripts.helpers import env_helpers, logging_helpers
+from dmscripts.helpers.html_helpers import render_html
+from dmscripts.helpers.logging_helpers import logging
+from dmutils.email.dm_mandrill import send_email
+from dmutils.formats import DATETIME_FORMAT
+
+
+EMAIL_SUBJECT = "New question and answer responses on the Digital Marketplace"
```

Comment:
For some reason the content doc for this is private., but this subject line doesn't make any sense to me.  What are "answer responses"?  

Either "New question responses..." or (better) "New answers..." seem better to me.

~~It would be nice to customise the subject to include the title (or at least reference number) of the brief rather than be generic (I imagine getting six emails a day with the exact same subject line will look pretty odd/crappy/spammy).~~

OK so it's one email with possibly multiple briefs listed.  Still, I don't think the subject line is right.